packages/amazonq/src/extensionNode.ts

```diff
@@ -53,9 +54,25 @@ async function activateAmazonQNode(context: vscode.ExtensionContext) {
     }
 
     if (!Experiments.instance.get('amazonqChatLSP', false)) {
+        const appInitContext = DefaultAmazonQAppInitContext.instance
+        const provider = new AmazonQChatViewProvider(
+            context,
+            appInitContext.getWebViewToAppsMessagePublishers(),
+            appInitContext.getAppsToWebViewMessageListener(),
+            appInitContext.onDidChangeAmazonQVisibility
+        )
+        context.subscriptions.push(
+            vscode.window.registerWebviewViewProvider(AmazonQChatViewProvider.viewType, provider, {
+                webviewOptions: {
+                    retainContextWhenHidden: true,
+                },
+            })
+        )
+        // this is registered inside of lsp/chat/activation.ts when the chat experiment is enabled
         await activateCWChat(context)
-        await activateQGumby(extContext as ExtContext)
     }
+    void registerAgents()
```

Comment:
What purpose does the void serve here? I've only seen this to avoid awaiting a promise but `registerAgents` isn't aync.

Reply:
It was just left over from when it was async